src/datepicker.js

```diff
@@ -1196,8 +1196,10 @@ function oneHandler(e) {
   */
   if (e.__qs_shadow_dom) return
 
+  var keyCode = e.keyCode
```

Comment:
One last thing... can you change this to: `var keyCode = e.which || e.keyCode`? Then you can can refactor the `if` statements to check a single variable. Once you make the change, I'll merge.

Thank you for the PR!

Reply:
Good point.